### PR TITLE
[#175476527] release pipeline

### PR DESCRIPTION
**Pull request by balanza**
Same changes applied in https://github.com/pagopa/io-functions-services/pull/96.

### Changes in brief
* Introduce the `Release` stage on the pipeline which marks a release on the repository
* Refactor the whole CI/CD to have 
  * separated pipelines for code review and deploy scenarios
  * everything is into `.devops` folder now

### CODE REVIEW

**Comment by balanza** [comment]
Wait for https://github.com/pagopa/io-developer-portal-frontend/pull/153 to close

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+# the build and deploy.
+pool:
+  vmImage: 'windows-2019'
+
+resources:
+  repositories:
+    - repository: pagopaCommons
+      type: github
+      name: pagopa/azure-pipeline-templates
+      ref: refs/tags/v3
```
Comment (pasqualedevita) [inline .devops/deploy-pipelines.yml:34]:
```suggestion
      ref: refs/tags/v4
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Pull request merged**